controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -0,0 +1,198 @@
+package controllers
```

Comment:
Controllers are really hard to review since it is not clear what was changed after they were moved. Could you move it first and then make changes in another commit?

Reply:
We don't need a different commit, at github is clearly stated that they have being moved from one place to the other.

Reply:
All the moved files are identical to their originals?

Reply:
moving or not moving the diff would be the same, there is no other option of moving place.

Reply:
If you first move it as is to a new directory and only in a next commit make needed changes to the file, it would be way easier to tell what is the difference and look for issues. In the current state, I have to review the whole controller despite only parts of it were modified.

Reply:
We are getting the difference already, I am still don't get what you mean.